csympy/lib/csympy_wrapper.pyx.in

```diff
@@ -6,6 +6,12 @@ from libcpp.string cimport string
 from cpython cimport PyObject, Py_XINCREF, Py_XDECREF, \
     PyObject_CallMethodObjArgs
 
+DEF HAVE_SAGE = ${HAVE_SAGE}
+have_sage = ${HAVE_SAGE}
```

Comment:
How about having a small config.pyx.in file, that defines HAVE_SAGE, and then we cimport it into csympy.pyx?